go.mod

```diff
@@ -56,61 +55,56 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
-require github.com/sigstore/cosign v1.5.2-0.20220302132634-73235ac02e4a
+require (
+	github.com/in-toto/in-toto-golang v0.3.4-0.20211211042327-af1f9fb822bf
```

Comment:
any reason for moving in-toto-golang to the next require section?

Reply:
Weird,  I can't reproduce it with mod tidy, but it is fixed now